### DISCUSSION TITLE
Web Wallet vs mobile wallet

### DISCUSSION THREAD

**Discussion by SethiShivam** [walt-id/.github#15] [Help] [answered]
Hi Team ,

I have a requirement where I need to handle say 5000  credentials in wallet and even more . 

I wanted to know if walt id web wallet will be able to handle that much stress. 
and which will be a better option for this much stress of credentials  , a web wallet or a mobile wallet. 
and why ?
@taminobaumann  @dombrn @philpotisk 

Best Regards
Sethi Shivam

**Comment by philpotisk** [comment, marked as answer]
Hi @SethiShivam the web-wallet is actually a Custodial wallet, where the credentials are stored on the server-side from whoever is running the service. This means, that the hosting provider needs to take care of scaling this service reliably. But basically there should be no issue to get this setup right. Please just reach out to us in case you want to discuss your use case and corresponding hosting requirements in detail.

All the best,
Phil
